app/api/dao/task_comment.py

```diff
@@ -203,8 +203,5 @@ def delete_comment(user_id, _id, task_id, relation_id):
                 HTTPStatus.NOT_FOUND,
             )
 
-        if task_comment:
-            task_comment.delete_from_db()
-            return messages.TASK_COMMENT_WAS_DELETED_SUCCESSFULLY, HTTPStatus.OK
```

Comment:
Nice catch :) you removed this because there is already a check up there

Reply:
Yes, thanks :)